utils/scripts/gen_tflite_models.py

```diff
@@ -288,21 +304,25 @@ def gen_unary_operator_test(name, type, input_shape):
     model = Model(inputs=[inp], outputs=[out])
     # Create data.
     np.random.seed(0)
-    inp_tensor = np.random.randn(*input_shape).astype(np.float32)
+    
+    inp_tensor = np.random.rand(*input_shape).astype(np.float32)
+    global representative_images
+    representative_images = inp_tensor
+    
     if type in ["log", "sqrt", "rsqrt"]:
         inp_tensor = np.abs(inp_tensor) + 1
     out_tensor = model.predict(inp_tensor)
     # Save model.
-    save_model(model, name)
+    save_model(model, name, quant)
     # Save data.
     save_tensor(inp_tensor, name + ".inp0")
     save_tensor(out_tensor, name + ".out0")
     # Clear session.
     keras_backend.clear_session()
 
-
+gen_unary_operator_test(name="hardSwish", type="hardSwish", input_shape=(1, 10))
```

Comment:
I would prefer renaming this model to "hard_swish" (as you see I wanted to be very consistent in terms of naming for the TFLite models ... I would prefer to have this cleaner than ONNX which is a real mess).